emcc

```diff
@@ -750,6 +755,16 @@ try:
   # counter for the next index that should be used.
   next_arg_index = len(newargs)
 
+  if pthreads:
+    settings_changes.append('USE_PTHREADS=1')
+    if not any(s.startswith('PTHREAD_POOL_SIZE=') for s in settings_changes):
+      settings_changes.append('PTHREAD_POOL_SIZE=0')
+    js_libraries.append(shared.path_from_root('src', 'library_pthread.js'))
+  else:
+    js_libraries.append(shared.path_from_root('src', 'library_pthread_stub.js'))
```

Comment:
Hmm actually, it might not. I'll have to do tests on that. For now I documented that compilation stage should do `-s USE_PTHREADS=1`, and linking stage should do `-lpthread`.